src/Nerdbank.MessagePack/StandardVisitor.cs

```diff
@@ -44,6 +45,7 @@ internal class StandardVisitor : TypeShapeVisitor, ITypeShapeFunc
 		{ typeof(byte[]), ByteArrayConverter.Instance },
 		{ typeof(Memory<byte>), new MemoryOfByteConverter() },
 		{ typeof(ReadOnlyMemory<byte>), new ReadOnlyMemoryOfByteConverter() },
+		{ typeof(Color), new SystemDrawingColorConverter() },
```

Comment:
The downside of doing this is that the library is now always roots System.Drawing components, which should negatively impact trimmed application size. We might want to investigate better techniques that help the linker to its job properly. One possibility that I haven't verified is replacing the dictionary of base converter types with a function like so:
```C#
static Converter<T>? GetBaseCaseConverter<T>()
{
    if (typeof(T) == typeof(System.Drawing.Color)
    {
         return (Converter<T>)(object)new SystemDrawingColorConverter();
    }

    /* more base case converter types */

    return null;
}
```
_Assuming_ the linker can eliminate branches for the `T` types it knows are not being exercised, then it would never be used unless opted in by the user.

cc @eerhardt